.github/workflows/node.js.yml

```diff
@@ -0,0 +1,33 @@
+name: Node.js CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x, 15.x]
```

Comment:
Our minimum Node version will bump up to 12.x in https://github.com/mozilla/protocol/pull/691, so we can drop 10.x from this list.